Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -453,39 +447,43 @@ def parse_json_files(filenames)
   end
 
   def merge_json_files(json_files)
-    json_files.reduce({}) do |hash, json_file|
-      hash.deep_merge(json_file) do |key, first, second|
-        if key == "cellar"
-          # Prioritize HOMEBREW_CELLAR over :any over :any_skip_relocation
-          cellars = [first, second]
-          next HOMEBREW_CELLAR if cellars.include?(HOMEBREW_CELLAR)
-          next first if first.start_with?("/")
-          next second if second.start_with?("/")
-          next "any" if cellars.include?("any")
-          next "any_skip_relocation" if cellars.include?("any_skip_relocation")
-        end
+    bottles_hashes = {}
+    json_files.each do |json_content|
+      formula_name = json_content.keys.first
 
-        second
+      jsc = json_content[formula_name]
```

Comment:
```suggestion
      formula_json = json_content[formula_name]
```

Reply:
Same here, this will be reworked in the second PR.